compiler/env/ExceptionTable.cpp

```diff
@@ -50,7 +50,7 @@ TR_ExceptionTableEntryIterator::TR_ExceptionTableEntryIterator(TR::Compilation *
       catchBlock = tt->getNode()->getBlock();
 
       // Skip OSR blocks
-      if (catchBlock->isOSRCodeBlock() || catchBlock->isOSRCatchBlock())
+      if (catchBlock->isOSRCodeBlock() || catchBlock->isOSRCatchBlock() || catchBlock->isOSRInduceBlock())
```

Comment:
I guess this is okay for now since we may never be inducing OSR in a catch block (?) but maybe this could be a place for an assertion to avoid having to debug a nasty exception table related problem in case someone did add an OSR induction in a catch block in the future (?)